src/content/contributing/building_testing/_index.en.md

```diff
@@ -45,29 +47,60 @@ To run Go unit tests:
 make test
 ```
 
-### End-to-end tests
+### Multi-Cluster KIND Based Environment {#clusters}
```

Comment:
oh cool  I had to look what {#clusters} was for. I see , adding an specific ID to the section link :)